src/main/kotlin/no/nav/klage/oppgave/domain/klage/Ankebehandling.kt

```diff
@@ -0,0 +1,95 @@
+package no.nav.klage.oppgave.domain.klage
+
+import no.nav.klage.kodeverk.Fagsystem
+import no.nav.klage.kodeverk.Type
+import no.nav.klage.kodeverk.Ytelse
+import no.nav.klage.kodeverk.hjemmel.Registreringshjemmel
+import no.nav.klage.kodeverk.hjemmel.RegistreringshjemmelConverter
+import no.nav.klage.oppgave.domain.Behandling
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+import javax.persistence.*
+
+@Entity
+@DiscriminatorValue("anke")
+class Ankebehandling(
+    @Column(name = "dato_vedtak_klageinstans")
+    val datoVedtakKlageinstans: LocalDate,
+    @Column(name = "avsender_enhet_klageinstans")
+    val avsenderEnhetKlageinstans: String,
```

Comment:
avsenderEnhet er ikke helt riktig navngiving her synes jeg. For klager, så har de jo først blitt behandlet av førsteinstans, og så sender de den videre til ka. Men for anker, så er ka første mottaker. Det vi er ute etter å fange her er hvilken enhet som gjorde vedtaket.

Reply:
Ja, enig i det du sier, jeg skal endre